diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/DiktatRuleSetFactoryImpl.kt

```diff
@@ -117,9 +117,9 @@ class DiktatRuleSetFactoryImpl : DiktatRuleSetFactory {
             // comments & documentation
             ::CommentsRule,
             ::SingleConstructorRule,  // this rule can add properties to a primary constructor, so should be before KdocComments
-            ::KdocComments,
             ::KdocMethods,
             ::KdocFormatting,
+            ::KdocComments,
```

Comment:
the problem is not in ordering:
1. We check KDoc when process `CLASS`
2. We fix the KDoc when process `VAL_KEYWORD` or `VAR_KEYWORD` inside.

We can move checking KDoc on CLASS level after processing the childs. But probably the fixing of KDoc for class should be done on class level

Reply:
created an issue for it #1776